graph_weather/data/IFSAnalysis_dataloader.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+import torchvision.transforms as transforms
+import xarray as xr
+from torch.utils.data import Dataset
+
+IFS_MEAN = {
+    "geopotential": 78054.78,
+    "specific_humidity": 0.0018220816,
+    "temperature": 243.41727,
+    "u_component_of_wind": 7.3073797,
+    "v_component_of_wind": 0.032221083,
+    "vertical_velocity": 0.0058287205,
+}
+
+IFS_STD = {
+    "geopotential": 59538.875,
+    "specific_humidity": 0.0035489395,
+    "temperature": 29.211119,
+    "u_component_of_wind": 13.777036,
+    "v_component_of_wind": 8.867598,
+    "vertical_velocity": 0.08577341,
+}
+
+
+class IFSAnalisysDataset(Dataset):
+    def __init__(self, start_year=2016, end_year=2022):
+        super().__init__()
+        self.data = xr.open_zarr(
+            "gs://weatherbench2/datasets/hres_t0/2016-2022-6h-64x32_equiangular_conservative.zarr"
```

Comment:
Could this be an input parameter? Incase we want to train on different resolutions?